src/main/ruby/core/method.rb

```diff
@@ -24,4 +24,6 @@ def curry(curried_arity = nil)
   
   alias_method :to_s, :inspect
 
+  alias_method :===, :call
```

Comment:
This is nice, but usually we try to keep aliases together for clarity.

Could you move this alias in the Java part in:
https://github.com/graalvm/truffleruby/blob/38f41992386db0500cde96bde98c3a44b37f9f5f/src/main/java/org/truffleruby/core/proc/ProcNodes.java#L213
?

Reply:
Much nicer, will do!

Reply:
I pointed you to the wrong place (Proc instead of Method), but you found the right place anyway :)